README.md

```diff
@@ -25,19 +25,26 @@ Learn more about Account Security and when to use the Authy API vs the Verify AP
 - Phone Verification
 - SMS or Voice Call
 
+### How to get an Authy API Key
+You will need to create a new Authy application in the [console](https://www.twilio.com/console/authy/). After you give it a name you can view the generated Account Security production API key. This is the string you will later need to set up in your environmental variables.
+
+![Get Authy API Key](api_key.png)
+
 ### Setup
 - Clone this repo
+- Create and activate a new virtual environment for this project
 - Run `pip install -r requirements.txt`
 - Register for a [Twilio Account](https://www.twilio.com/).
 - Setup an Account Security app via the [Twilio Console](https://twilio.com/console).
 - Grab an Application API key from the Dashboard and paste it in `.env.example`
```

Comment:
Can we include a screenshot here?

Reply:
Can you elaborate more in which step, please?
There's already a screenshot at the beginning regarding how to get the Authy API KEY.

Reply:
Perhaps #53 ?

Reply:
ah i missed that. that covers that step. 

but i think what i was pointing out was that this particular code sample seems to differ from the other ones which show the exact commands necessary for cloning, creating a virtual env etc. this isn't absolutely necessary, but it would be great if they all maintained that same template.

not a blocker for this PR though.